sources/packages/web/src/components/common/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -46,6 +46,8 @@
         :assessmentTriggerType="
           applicationProgressDetails.assessmentTriggerType!
         "
+        :applicationId="applicationId"
+        :assessmentId="applicationProgressDetails.currentAssessmentId"
```

Comment:
Instead of passing properties we can keep the component simple and make it just emit an event.